### PR TITLE
Add LocalResponseNorm and BatchNorm in training

### PR DESCRIPTION
**Pull request by jatinmandav**
Addition of choice between LocalResponseNormalization and BatchNormalization and command line based control of train.py file

### CODE REVIEW

**Comment by Borda** [comment]
@jatinmandav thank you for your suggestion, may you first resolve conflicts to be aligned with actual master... Thx

**Comment by jatinmandav** [comment]
I will resolve and let you know :)

**Comment by Borda** [comment]
@jatinmandav pls, set your "upstream" to point to this repository and rebase it
```
git remote add upstream https://github.com/Borda/keras-yolo3.git
git rebase upstream/master
```

File: README.md [hunk 1]
```diff
@@ -69,7 +69,32 @@ optional arguments:
 3. Modify train.py and start training.  
     `python train.py`  
     Use your trained weights or checkpoint weights with command line option `--model model_file` when using yolo_video.py
-    Remember to modify class path or anchor path, with `--classes class_file` and `--anchors anchor_file`.
+    Remember to modify class path or anchor path, with `--classes class_file` and `--anchors anchor_file`. Use `--use_lrn` flag to use LocalResponseNormalization layer instead of BatchNormalization
+    
+    ```
+    python3 train.py --help
```
Comment (Borda) [inline README.md]: I do not think that this is needed since it is printed as help for each script

File: train.py [hunk 2]
```diff
@@ -73,14 +86,14 @@ def _main():
         model.compile(optimizer=Adam(lr=1e-4), loss={'yolo_loss': lambda y_true, y_pred: y_pred}) # recompile to apply the change
         print('Unfreeze all of the layers.')
 
-        batch_size = 32 # note that more GPU memory is required after unfreezing the body
+        batch_size = 16 # note that more GPU memory is required after unfreezing the body
```
Comment (Borda) [inline train.py]: most of the configuration was moved to a special file

File: yolo3/LRN.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
+from keras.layers.core import Layer
```
Comment (Borda) [inline yolo3/LRN.py]: name the file with lower characters and maybe longer name

**Comment by jatinmandav** [comment]
I have re-coded with this repo as base, and have created a new pull request so am closing this.
